bundle/manifests/flows.netobserv.io_flowcollectors.yaml

```diff
@@ -0,0 +1,326 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Is this the same file than config/crd/bases/flows.netobserv.io_flowcollectors.yaml

Is there a way to avoid duplicate file ?

Reply:
This is automatically generated by the operator-sdk and `bundle` make targets. I guess we could but do something but we would break the "standard" workflow.